info/v2/container.go

```diff
@@ -146,6 +146,8 @@ type ContainerStats struct {
 	Filesystem *FilesystemStats `json:"filesystem,omitempty"`
 	// Task load statistics
 	Load *v1.LoadStats `json:"load_stats,omitempty"`
+	// RestartCount contains how many times a container was restarted automatically
+	RestartCount int `json:"restartcount,omitempty"`
```

Comment:
[This is where we convert.  Should be simple to implement](https://github.com/google/cadvisor/blob/master/info/v2/conversion.go).